FrontEnd/src/App.js

```diff
@@ -63,8 +66,20 @@ class App extends React.Component {
             setStudy={this.setSelectedStudy} 
             setStudies={this.setStudies} 
             setArchived={this.setArchived} /> : null}
-        {this.state.page === "study" ? <CreateStudyPage state={this.state} changePage={this.changePage} setStudy={this.setSelectedStudy} /> : null}
-        {this.state.page === "pod" ? <PodPage state={this.state} changePage={this.changePage} setStudy={this.setSelectedStudy} /> : null}
+
+        {this.state.page === "study" ? 
+          <CreateStudyPage 
+            state={this.state} 
+            changePage={this.changePage} 
+            setStudy={this.setSelectedStudy}
+            setSavingState={this.setSavingState} /> : null}
+            
+        {this.state.page === "pod" ? 
+          <PodPage 
+            state={this.state} 
+            changePage={this.changePage} 
+            setStudy={this.setSelectedStudy}
+            setSavingState={this.setSavingState} /> : null}
```

Comment:
When you go multi line i recommend using if statements over ?:. ?: is not the most readable of syntax.